### ISSUE TITLE
can't get google drive access . but i tried google sign but it works  

### ISSUE THREAD

**Issue by rohitashyogi0507620** [mesadhan/google-drive-app#8]
hey can't able to get drive account . when I select account it buffer a loader but noting happened 
look at this screen shot please help me out 

![google drive error](https://user-images.githubusercontent.com/65347250/103973767-433dcd00-5196-11eb-86b6-25fc78de8a35.png)

**Comment by Automage** [comment]
Have the same problem.

EDIT: Encountered this problem as well. Looks like it has something to do with the scopes being requested. Requesting DRIVE.READONLY causes this problem for me.

**Comment by RohitSurwase** [comment]
For me it happens, irrespective of which permission I choose to access.

**Comment by whhdxl** [comment]
So do I ， does anyone solved it?

**Comment by rohitashyogi0507620** [comment]
Yes i just solved it . we need to add  Scopes in oAuth consent screen in drive api console  when  Verification added in drive api 

![Screenshot (37)](https://user-images.githubusercontent.com/65347250/107461758-b0bb8f80-6b80-11eb-881d-68eceeab6933.png)

**Comment by RohitSurwase** [comment]
Hi @rohitashyogi0507620 , I had already added scope using OAuth consent screen still I have the issue. Could you please list down all steps (So I would know what exactly I missed.) required to make this working? I would really appreciate that.

**Comment by chintandesai49** [comment]
@rohitashyogi0507620 I have added all 5 permission which you have shown. It would be very helpful if you can you show all 4 OAuth Consent screens, and which credentials you have added to the console.

**Comment by oshin5057** [comment]
@rohitashyogi0507620, this problem was also showing for me.
But I solved this problem.
You need to add a G-mail id in the OAuth consent screen with you want to log in and access the drive.
Follow these steps:
After creating OAuth client ID, you need to click ADD Users button,
![image](https://user-images.githubusercontent.com/61081148/131238333-53efecb8-acd5-422a-b73c-4dd599666887.png)
then add your g-mail id,
then rebuild your project, and then you can log in to it using your G-mail ID which you have added to the OAuth Client ID.
